### PR TITLE
Tsk 1021

### PR DESCRIPTION
**Pull request by BerndBreier**

### CODE REVIEW

File: lib/taskana-core/src/main/java/pro/taskana/mappings/WorkbasketMapper.java [hunk 1]
```diff
@@ -25,8 +27,8 @@
       value = {
         @Result(property = "id", column = "ID"),
         @Result(property = "key", column = "KEY"),
-        @Result(property = "created", column = "CREATED"),
-        @Result(property = "modified", column = "MODIFIED"),
+        @Result(property = "created", column = "CREATED", jdbcType = JdbcType.TIMESTAMP, javaType = java.time.Instant.class, typeHandler = InstantTypeHandler.class),
```
Comment (benjamineckstein) [inline lib/taskana-core/src/main/java/pro/taskana/mappings/WorkbasketMapper.java]: i dont like that. too much risk of forgetting any "timestamp" field. Think about new ones in the future added by someone else ...

Comment (BerndBreier) [reply lib/taskana-core/src/main/java/pro/taskana/mappings/WorkbasketMapper.java]: That's correct. But do you have a better alternative?

**Comment by mustaphazorgati** [comment]
Hey @BerndBreier,

looks like something went wrong. You got all your commits twice and you have a merge within the same branch. Let's check this out together tomorrow.

**Comment by mustaphazorgati** [comment]
And also remove the changes to the package-lock file

**Comment by BerndBreier** [comment]
Please don't merge this PR. We are currently trying to convince Mybatis to use the statements from our InstantTypeHandler in their InstantTypeHandler. This would simplify the code enormously because no annotation would need to be changed ...
Only if Mybatis refuses, we should use the solution proposed by this PR

**Comment by BerndBreier** [comment]
Found a better way to fix the problem. If the type handler is registered >>before<< the mappers are registered, it is used automatically and no annotations are required

**Pull request closed without merging**